parameters.settings.json

```diff
@@ -0,0 +1,552 @@
+            "group": "Trimming presets",
+            "render": "check-box",
+            "default_value": false,
+            "type": "boolean"
+        },
+        {
+            "name": "single_end",
+            "label": "Single-end sequencing input",
+            "usage": "Use single-end sequencing inputs instead of paired-end.",
+            "group": "Main options",
+            "render": "check-box",
+            "default_value": false,
+            "type": "boolean"
+        },
+        {
+            "name": "clip_r1",
+            "label": "Read Clipping: 5' R1",
+            "usage": "Instructs Trim Galore to remove bp from the 5' end of read 1 (or single-end reads).",
+            "group": "Read trimming",
+            "render": "textfield",
+            "pattern": "\\d*",
+            "type": "integer",
+            "default_value": 0
+        },
+        {
+            "name": "clip_r2",
+            "label": "Read Clipping: 5' R1",
+            "usage": "Instructs Trim Galore to remove bp from the 5' end of read 2 (paired-end reads only).",
+            "group": "Read trimming",
+            "render": "textfield",
+            "pattern": "\\d*",
+            "type": "integer",
+            "default_value": 0
+        },
+        {
+            "name": "three_prime_clip_r1",
+            "label": "Read Clipping: 3' R1",
+            "usage": "Instructs Trim Galore to remove bp from the 3' end of read 1 AFTER adapter/quality trimming has been performed.",
+            "group": "Read trimming",
+            "render": "textfield",
+            "pattern": "\\d*",
+            "type": "integer",
+            "default_value": 0
+        },
+        {
+            "name": "three_prime_clip_r2",
+            "label": "Read Clipping: 3' R2",
+            "usage": "Instructs Trim Galore to re move bp from the 3' end of read 2 AFTER adapter/quality trimming has been performed.",
+            "group": "Read trimming",
+            "render": "textfield",
+            "pattern": "\\d*",
+            "type": "integer",
+            "default_value": 0
+        },
+        {
+            "name": "aligner",
+            "label": "Alignment tool",
+            "usage": "Choose whether to align reads with bismark (Bowtie2), bismark_hisat (HISAT2) or bwameth(BWA)",
+            "type": "string",
+            "render": "radio-button",
+            "choices": [
+                    "bismark",
+                    "bismark_hisat",
+                    "bwameth"
+            ],
+            "default_value": "bismark",
+            "group": "Alignment"
+        },
+        {
+            "name": "genome",
+            "label": "Alignment reference iGenomes key",
+            "usage": "Ref. genome key for iGenomes",
+            "group": "Alignment",
+            "render": "drop-down",
+            "type": "string",
+            "choices": [
+                    "",
+                    "GRCh37",
+                    "GRCm38",
```

Comment:
GRch38 is missing here right? I saw that it was added to the igenomes config file.

Reply:
Added in 8462b79